packit_service/worker/pr_comment_handler.py

```diff
@@ -28,22 +28,22 @@
 import logging
 import shutil
 
-from typing import Dict, Type, Optional
+from typing import Dict, Type, Optional, Union
 from pathlib import Path
 
 from ogr import GithubService
 from packit.api import PackitAPI
 from packit.config import Config
 
-from packit_service.service.events import PullRequestCommentEvent
+from packit_service.service.events import PullRequestCommentEvent, IssueCommentEvent
 from packit_service.worker.handler import HandlerResults
 
 logger = logging.getLogger(__name__)
 
 
 class PullRequestCommentAction(enum.Enum):
```

Comment:
Could this be just `CommentAction` as it can be both the issue and PR?

Reply:
Good proposal.